docs/configuration.md

```diff
@@ -51,13 +51,14 @@ following network protocols (net part of the resolver URL):
     returns the answer from the fastest one. This improves your network speed and increases your privacy - your DNS traffic
     will be distributed over multiple providers.
 
-Each resolver must be defined as a string in following format: `[net:]host:[port][/path]`.
+Each resolver must be defined as a string in following format: `[net:]host:[port][/path][#commonName]`.
 
 | Parameter | Type                             | Mandatory | Default value                                     |
 |-----------|----------------------------------|-----------|---------------------------------------------------|
-| net       | enum (tcp+udp, tcp-tls or https) | no        | tcp+udp                                           |
-| host      | IP or hostname                   | yes       |                                                   |
-| port      | int (1 - 65535)                  | no        | 53 for udp/tcp, 853 for tcp-tls and 443 for https |
+| net        | enum (tcp+udp, tcp-tls or https) | no        | tcp+udp                                           |
+| host       | IP or hostname                   | yes       |                                                   |
+| port       | int (1 - 65535)                  | no        | 53 for udp/tcp, 853 for tcp-tls and 443 for https |
+| commonName | string                           | no        | the host value                                    |
```

Comment:
I think the common name's description should make it explicit it's used for cert checking.  
Maybe "the expected certificate common name (host) value"

Reply:
The last column is default value? I've added prose explaining the semantics of the field.

Reply:
Ah yeah my bad!